xml/containers-docker-dockerizing-application.xml

```diff
@@ -356,15 +354,15 @@ CMD ["-D", "FOREGROUND"]</screen>
     </callout>
     <callout arearefs="co-dockerfile-apache2-echo">
      <para>
-      Test line for debugging purposes, can be removed if everything
-      works as expected.
+      Test line for debugging purposes, can be removed if everything works as
```

Comment:
```suggestion
      Test line for debugging purposes; can be removed if everything works as
```

Reply:
Please don't suggest changes that are against the style guide. A period would make sense of course.